net/net.go

```diff
@@ -23,7 +23,5 @@ type Neter interface {
 
 func StartProtocol(pubKey *crypto.PubKey) protocol.Noder {
 	net := node.InitNode(pubKey)
-	net.ConnectSeeds()
```

Comment:
The function be called here to speed up the connection process, which means the initialization connection don't need wait 3 second until the update info timer be triggered